src/app/login/login-page/login-page.component.ts

```diff
@@ -3,16 +3,13 @@ import { OAuthService } from 'angular-oauth2-oidc';
 
 @Component({
   selector: 'app-login-page',
-  templateUrl: './login-page.component.html',
-  styleUrls: ['./login-page.component.sass']
```

Comment:
Looks like the file itself was not removed.

Reply:
fixed